lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@poliva83 given the above, I believe the best course of action is to create a raw_rest in the base provider (next to https://github.com/chef/cheffish/blob/master/lib/cheffish/chef_provider_base.rb#L7-L9) that looks like this:

```ruby
    def raw_rest
      @raw_rest ||= Cheffish.chef_server_api(new_resource.chef_server.merge(options: { raw_output: true }))
    end
```

And use that for the `post` request, so that it doesn't parse the output.